msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -112,13 +112,17 @@ The following image illustrates the in-meeting side panel:
 
 ---
 
-The following table describes the behavior of app when it's validated and not validated:
+The following table describes the behavior of app in channel meetings.
 
 |App capability | App is validated | App isn't validated |
-|---|---|---|
-| Meeting extensibility | The app will appear in meetings. | The app won't appear in meetings for the mobile clients. |
+|---|---|---||
+| Allows user to set up a meeting from a channel |  users can use apps in scheduled channel meetings. | External members are not allowed to participate in channel meetings.Permission is required |
```

Comment:
1. are not replace with aren't
2. add space before premissions

Reply:
done